### PR TITLE
[SYCL][DOC] Collecting open issues

### PR DESCRIPTION
**Pull request by reble**
This patch adds gathered feedback, including feedback from https://github.com/intel/llvm/pull/5626
to the issues section. Including a trend for each feature.

### CODE REVIEW

File: sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc [hunk 1]
```diff
@@ -1525,7 +1526,35 @@ the allocation. No mechanism is currently provided, but see the
 section on <<memory-allocation-nodes, Memory Allocation Nodes>> for
 some designs being considered.
 
-**Outcome:** Designs under consideration
+**UNRESOLVED:** Trending "yes". Design is under consideration
```
Comment (EwanC) [inline sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc]:
```suggestion
**UNRESOLVED:** Trending "yes". Design is under consideration.
```

File: sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc [hunk 2]
```diff
@@ -1525,7 +1526,35 @@ the allocation. No mechanism is currently provided, but see the
...
+
+=== Device agnostic Graph
```
Comment (EwanC) [inline sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc]:
```suggestion
=== Device Agnostic Graph
```

File: sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc [hunk 3]
```diff
@@ -1525,7 +1526,35 @@ the allocation. No mechanism is currently provided, but see the
...
+
+Explicit API could support devcie-agnostic graphs that can be submitted
```
Comment (EwanC) [inline sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc]:
```suggestion
Explicit API could support device-agnostic graphs that can be submitted 
```

Also, i'm not sure if just Explicit API could support this. Record & Replay could do device agnostic graphs by ignoring the device associated with the queue.

File: sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc [hunk 4]
```diff
@@ -1525,7 +1526,35 @@ the allocation. No mechanism is currently provided, but see the
...
+=== Device agnostic Graph
+
+Explicit API could support devcie-agnostic graphs that can be submitted 
+through queues to a particular device. This issue is related to multi-device 
+graphs.
+
+**UNRESOLVED:** Trending "no". Because of current runtime limitations this 
+can't be implemented with a reasonable effort.
+
+=== Execution property
```
Comment (EwanC) [inline sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc]:
```suggestion
=== Execution Property
```

File: sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc [hunk 5]
```diff
@@ -1525,7 +1526,35 @@ the allocation. No mechanism is currently provided, but see the
...
+
+Current proposal contains extensive extensions to existing API in SYCL.
+Can we achieve something similar with user control over the flush behavior
+of a queue and providing a handler that can be replayed?
+
+**UNRESOLVED:** Trending "no". Needs reconsideration of the design and
+possible restrictions.
+
+=== User guided scheduling
```
Comment (EwanC) [inline sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc]:
```suggestion
=== User Guided Scheduling
```

File: sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc [hunk 6]
```diff
@@ -1525,7 +1526,35 @@ the allocation. No mechanism is currently provided, but see the
...
+
+For specific workloads it could be beneficial to provide hints to the
+runtime how to schedule a command graph onto a device. This info could effect
+the scheduling policy like breadth or depth-first, or a combination with a
+block size.
+
+**UNRESOLVED:** Trending "yes". A new property could be added to 
+the finalize call.
```
Comment (EwanC) [inline sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc]: Would mention the layered extension idea here too.

Comment (reble) [inline sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc]:
```suggestion
**UNRESOLVED:** Trending "yes". A new property could be added to 
the finalize call either extending the basic command graph proposal
or layered as a separate extension proposal. 
```

**Review by EwanC — APPROVED** [review]
LGTM 🙂

**Review by Bensuo — APPROVED** [review]
LGTM

**Pull request merged**